### PR TITLE
django 2.0 support

### PR DESCRIPTION
**Pull request by andrewshkovskii**
This PR make available to use `pyuploadcare` with django 2.0
Version 1.11 will be also supported.

REFS: #131

### CODE REVIEW

**Comment by andrewshkovskii** [comment]
[pyuploadcare on django 2.0](https://trello.com/c/rlbO30i0/61-pyuploadcare-on-django-20)

File: HISTORY.rst [hunk 1]
```diff
@@ -6,6 +6,12 @@ History
 2.2.1
 ~~~~~
```
Comment (dmitry-mukhin) [inline HISTORY.rst]: 2.3.0?

File: HISTORY.rst [hunk 2]
```diff
@@ -6,6 +6,12 @@ History
...
 
+- Add support for django versions 1.11 and ~2.0
+- Miscellaneous updates(version, year in a licence file, tox configuration and etc.)
```
Comment (dmitry-mukhin) [inline HISTORY.rst]: license

File: README.rst [hunk 3]
```diff
@@ -105,6 +105,7 @@ Contributors
 - `@zerc`_
 - `@homm`_
 - `@va1en0k`_
+- `@andreshkovskii`_
```
Comment (dmitry-mukhin) [inline README.rst]: 👍

**Comment by dmitry-mukhin** [comment]
also, add py36 to both tox and travis

File: tox.ini [hunk 4]
```diff
@@ -8,6 +8,7 @@ basepython =
     py33: python3.3
     py34: python3.4
     py35: python3.5
+    py36: python3.5
```
Comment (dmitry-mukhin) [inline tox.ini]: O_o

File: .travis.yml [hunk 5]
```diff
@@ -6,13 +6,15 @@ python:
...
   - 3.4
   - 3.5
+  - 3.6
 env:
   global:
     - SUITE="tests/functional"
   matrix:
     - DJANGO=1.8
     - DJANGO=1.9
     - DJANGO=1.10
```
Comment (dmitry-mukhin) [inline .travis.yml]: add 1.11

File: .travis.yml [hunk 6]
```diff
@@ -6,13 +6,15 @@ python:
   - 2.7
```
Comment (dmitry-mukhin) [inline .travis.yml]: probably, this should be moved to include as dj2 works only with py3

File: .travis.yml [hunk 7]
```diff
@@ -30,6 +32,9 @@ matrix:
     - python: 3.3
       env: DJANGO=1.8
 
+    - python: 3.6
+      env: DJANGO=2.0
```
Comment (dmitry-mukhin) [inline .travis.yml]: this is redundant, as it's already added with python+matrix settings

**Comment by dmitry-mukhin** [comment]
#133

**Pull request merged**